recipes/jupyter-wxyz/meta.yaml

```diff
@@ -0,0 +1,399 @@
+{% set version = "0.4.1" %}
+{% set min_peer = "0.4" %}
+{% set max_peer = "0.5" %}
+{% set build_number = "0" %}
+
+package:
+  name: jupyter-wxyz
+  version: {{ version }}
+
+source:
+  - folder: wxyz_core
+    url: https://pypi.io/packages/source/w/wxyz_core/wxyz_core-{{ version }}.tar.gz
+    sha256: ad091a5e53cd17e93c82ac805d2114608f1071bb039cc8c56c44bc6004ff7eaa
+  - folder: wxyz_datagrid
+    url: https://pypi.io/packages/source/w/wxyz_datagrid/wxyz_datagrid-{{ version }}.tar.gz
+    sha256: 4f8addeb05e1fc719f69d51fc4c71df3fbf5a0df351a352c9b3a3198df80306a
+  - folder: wxyz_dvcs
+    url: https://pypi.io/packages/source/w/wxyz_dvcs/wxyz_dvcs-{{ version }}.tar.gz
+    sha256: a9f5982b6d274803ec13e2044278c8fe7236f68360bcb38c83fa578c3458a424
+  - folder: wxyz_html
+    url: https://pypi.io/packages/source/w/wxyz_html/wxyz_html-{{ version }}.tar.gz
+    sha256: 3362c634279fc41d2e94c69bc79db5faa1db5425fa0c1edf6b949703469c2bec
+  - folder: wxyz_json_e
+    url: https://pypi.io/packages/source/w/wxyz_json_e/wxyz_json_e-{{ version }}.tar.gz
+    sha256: bccfc0de7cc412efd8a91d3c087725d1a8ef4b64fe84be1ec6958b6b5d413612
+  - folder: wxyz_json_schema_form
+    url: https://pypi.io/packages/source/w/wxyz_json_schema_form/wxyz_json_schema_form-{{ version }}.tar.gz
+    sha256: fe0589b69e7d445ec10b60a2be23315d17a6acc02b6afa6873967ba478f3fe4b
+  - folder: wxyz_jsonld
+    url: https://pypi.io/packages/source/w/wxyz_jsonld/wxyz_jsonld-{{ version }}.tar.gz
+    sha256: cceab32476371bd462690ffd9c341acf0968e554118fc7c06430a0188c337d64
+  - folder: wxyz_lab
+    url: https://pypi.io/packages/source/w/wxyz_lab/wxyz_lab-{{ version }}.tar.gz
+    sha256: 3893572c3cc41fec8d6be022ea39e8fa68e74d42216b3f1dce733b9f340d39bd
+  - folder: wxyz_svg
+    url: https://pypi.io/packages/source/w/wxyz_svg/wxyz_svg-{{ version }}.tar.gz
+    sha256: 80fef008b497c50e6ce1666e8cb30a02f9a98305f4b65700b85a74cea720b0a9
+  - folder: wxyz_tpl_jinja
+    url: https://pypi.io/packages/source/w/wxyz_tpl_jinja/wxyz_tpl_jinja-{{ version }}.tar.gz
+    sha256: 61ff63848d53d2729a4ad3b1398ef4f888928977bea3e1ebc5a580de21c57cb6
+  - folder: wxyz_yaml
+    url: https://pypi.io/packages/source/w/wxyz_yaml/wxyz_yaml-{{ version }}.tar.gz
+    sha256: 68caa57d39007be2015369266db1a35355a32279c9664f8adaa40410d68fa62d
+
+build:
+  number: {{ build_number }}
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  commands:
+    - echo TBD
+
+outputs:
+  - name: wxyz_core
+    build:
+      number: {{ build_number }}
+      noarch: python
+      script: cd wxyz_core && {{ PYTHON }} -m pip install . -vv --no-deps
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+      run:
+        - ipywidgets
+        - jsonpointer
+        - python >=3.6
+    test:
+      requires:
+        - pip
+        - jedi <0.18
```

Comment:
Once https://github.com/conda-forge/jedi-feedstock/pull/32 is up on CDN, i'll drop all the jedi stuff...